website/docs/intro/install/standalone.mdx

```diff
@@ -1,39 +1,57 @@
 ---
 sidebar_position: 99
-sidebar_label: Standalone (Linux/MacOS/Windows)
+sidebar_label: Standalone (Linux/MacOS/Windows/BSD)
 description: |-
   Use OpenTofu as a standalone binary without installation.
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
-import CodeBlock from '@theme/CodeBlock';
-import LinuxScript from '!!raw-loader!./standalone-install.sh'
-import WindowsScript from '!!raw-loader!./standalone-install.ps1'
-import VerifyChecksumPS1 from '!!raw-loader!./verify-checksum.ps1'
-import VerifyChecksumLinux from '!!raw-loader!./verify-checksum-linux.sh'
-import VerifyChecksumMacOS from '!!raw-loader!./verify-checksum-macos.sh'
-import VerifyCosignLinux from '!!raw-loader!./verify-cosign.sh'
-import VerifyCosignWindows from '!!raw-loader!./verify-cosign.ps1'
-import Admonition from '@theme/Admonition';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import CodeBlock from "@theme/CodeBlock";
+import LinuxScript from "!!raw-loader!./standalone-install.sh";
+import WindowsScript from "!!raw-loader!./standalone-install.ps1";
+import VerifyChecksumPS1 from "!!raw-loader!./verify-checksum.ps1";
+import VerifyChecksumLinux from "!!raw-loader!./verify-checksum-linux.sh";
+import VerifyChecksumMacOS from "!!raw-loader!./verify-checksum-macos.sh";
+import VerifyCosignLinux from "!!raw-loader!./verify-cosign.sh";
+import VerifyCosignWindows from "!!raw-loader!./verify-cosign.ps1";
+import Admonition from "@theme/Admonition";
 
 # Installing OpenTofu from GitHub Releases
 
 ## Using the installer script
 
 <Tabs>
-    <TabItem value="posix" label="Linux/MacOS/Unix (POSIX)">
-        <CodeBlock language="bash">{LinuxScript}</CodeBlock>
-        <Admonition type="info">The standalone installer verifies the integrity of the downloaded files. You need to install <a href="https://docs.sigstore.dev/system_config/installation/">cosign</a>, <a href="https://gnupg.org/">GnuPG</a>, or disable the integrity verification by using the <code>--skip-verify</code> option.</Admonition>
-    </TabItem>
-    <TabItem value="powershell" label="Windows (PowerShell)">
-        <CodeBlock language="powershell">{WindowsScript}</CodeBlock>
-        <Admonition type="info">If you run into script execution policy issues when running this script, please run <code>Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process</code> before running the installer.</Admonition>
-        <Admonition type="info">The standalone installer verifies the integrity of the downloaded files. You need to install <a href="https://docs.sigstore.dev/system_config/installation/">cosign</a>, <a href="https://gnupg.org/">GnuPG</a>, or disable the integrity verification by using the <code>-skipVerify</code> option.</Admonition>
-    </TabItem>
+  <TabItem value="posix" label="Linux/MacOS/Unix (POSIX)">
```

Comment:
Why have these files been reformatted?

Reply:
It's vscode, do we have any guideline for formatting ?